app/assets/javascripts/angular/controller/home-controller.js.erb

```diff
@@ -22,9 +37,16 @@ app.controller('HomeCtrl', ['$scope', '$http', '$rootScope', function($scope, $h
       $scope.card.bestActressMovie = data.movie[0]["bestActressMovie"];
     })
     $http.get('songlists/' + year + '.json').success(function(data){
```

Comment:
Maybe you want to make this a separate function just like how you did it with `getHoroscopes`